### ISSUE TITLE
bug: stuck at "downloading" on macOS 10.15

### ISSUE THREAD

**Issue by MentalGear** [electron/fiddle#1455]
Computer for work are using 10.15 (as it is stable, and honestly better looking). However, the latest version of fiddle (which is a great idea) downloads, but gets then stuck at "Downloading", which makes it unusable.

**Comment by codebytere** [comment]
@MentalGear can you please explain a little more what the issue is? What is stuck at downloading? Individual versions of Electron? The app itself? Screenshots would be helpful.

**Comment by MentalGear** [comment]
Thx for the reply, here's what stuck.

<img width="273" alt="Screenshot 2023-09-07 at 12 22 05" src="https://github.com/electron/fiddle/assets/2837147/3dfa425e-376e-42d3-a0f2-58a0a4e51fc2">

**Comment by codebytere** [comment]
@MentalGear thanks! does this happen for every version, or only sometimes?

**Comment by MentalGear** [comment]
For any electron version I try to download. If you mean fiddle version, I only tried the latest.

Here's the mac console.app log:
```shell
default	12:46:17.207472+0200	electron-fiddle	Connection 1: is being canceled
default	12:46:17.207571+0200	electron-fiddle	[C1 07175D42-018A-4189-B234-B202E7A9167E Hostname#0665a351:443 tcp, url hash: d431586c, tls] cancel
default	12:46:17.208359+0200	electron-fiddle	[C1 07175D42-018A-4189-B234-B202E7A9167E Hostname#0665a351:443 tcp, url hash: d431586c, tls] cancelled
	[C1.1 858ABB99-3F95-4813-A4AE-74F0E8F7EE83 192.168.178.31:59190<->IPv4#ba1b28d9:443]
	Connected Path: satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns
	Duration: 60.585s, DNS @0.000s took 0.054s, TCP @0.055s took 0.258s, TLS took 0.272s
	bytes in/out: 4825/846, packets in/out: 8/3, rtt: 0.218s, retransmitted packets: 0, out-of-order packets: 0
default	12:46:17.208419+0200	electron-fiddle	0.000s [C1 1A35F8FB-2660-4B58-B624-D5C2FD6F36AF Hostname#0665a351:443 resolver path=satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns] path:start
default	12:46:17.208457+0200	electron-fiddle	0.000s [C1 1A35F8FB-2660-4B58-B624-D5C2FD6F36AF Hostname#0665a351:443 resolver path=satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns] path:satisfied
default	12:46:17.208491+0200	electron-fiddle	0.000s [C1 1A35F8FB-2660-4B58-B624-D5C2FD6F36AF Hostname#0665a351:443 resolver path=satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns] resolver:start_dns
default	12:46:17.208523+0200	electron-fiddle	0.054s [C1 1A35F8FB-2660-4B58-B624-D5C2FD6F36AF Hostname#0665a351:443 resolver path=satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns] resolver:receive_dns
default	12:46:17.208564+0200	electron-fiddle	0.054s [C1.1 858ABB99-3F95-4813-A4AE-74F0E8F7EE83 192.168.178.31:59190<->IPv4#ba1b28d9:443 socket-flow path=satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns] path:start
default	12:46:17.208596+0200	electron-fiddle	0.054s [C1.1 858ABB99-3F95-4813-A4AE-74F0E8F7EE83 192.168.178.31:59190<->IPv4#ba1b28d9:443 socket-flow path=satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns] path:satisfied
default	12:46:17.208628+0200	electron-fiddle	0.055s [C1.1 858ABB99-3F95-4813-A4AE-74F0E8F7EE83 192.168.178.31:59190<->IPv4#ba1b28d9:443 socket-flow path=satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns] flow:start_connect
default	12:46:17.208659+0200	electron-fiddle	0.260s [C1 1A35F8FB-2660-4B58-B624-D5C2FD6F36AF Hostname#0665a351:443 resolver path=satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns] resolver:receive_dns
default	12:46:17.208708+0200	electron-fiddle	0.313s [C1.1 858ABB99-3F95-4813-A4AE-74F0E8F7EE83 192.168.178.31:59190<->IPv4#ba1b28d9:443 socket-flow path=satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns] flow:finish_transport
default	12:46:17.208740+0200	electron-fiddle	0.313s [C1 1A35F8FB-2660-4B58-B624-D5C2FD6F36AF Hostname#0665a351:443 resolver path=satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns] flow:finish_transport
default	12:46:17.208774+0200	electron-fiddle	0.585s [C1.1 858ABB99-3F95-4813-A4AE-74F0E8F7EE83 192.168.178.31:59190<->IPv4#ba1b28d9:443 socket-flow path=satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns] flow:finish_connect
default	12:46:17.208805+0200	electron-fiddle	0.585s [C1 1A35F8FB-2660-4B58-B624-D5C2FD6F36AF Hostname#0665a351:443 resolver path=satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns] flow:finish_connect
default	12:46:17.208837+0200	electron-fiddle	0.585s [C1.1 858ABB99-3F95-4813-A4AE-74F0E8F7EE83 192.168.178.31:59190<->IPv4#ba1b28d9:443 socket-flow path=satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns] flow:changed_viability
default	12:46:17.208935+0200	electron-fiddle	0.585s [C1 1A35F8FB-2660-4B58-B624-D5C2FD6F36AF Hostname#0665a351:443 resolver path=satisfied (Path is satisfied), interface: en0, ipv4, ipv6, dns] flow:changed_viability
default	12:46:17.208976+0200	electron-fiddle	60.585s [C1] path:cancel
default	12:46:17.209310+0200	electron-fiddle	boringssl_context_message_handler(2258) [C1.1:2][0x7ff08d97d950] Writing SSL3_RT_ALERT 2 bytes
default	12:46:17.209345+0200	electron-fiddle	boringssl_context_handle_warning_alert(1893) [C1.1:2][0x7ff08d97d950] write alert, level: warning, description: close notify
default	12:46:17.209374+0200	electron-fiddle	boringssl_session_disconnect(504) [C1.1:2][0x7ff08d97d950] SSL_shutdown 0
default	12:46:17.209402+0200	electron-fiddle	nw_flow_disconnected [C1.1 IPv4#ba1b28d9:443 cancelled socket-flow ((null))] Output protocol disconnected
default	12:46:17.209671+0200	electron-fiddle	nw_connection_report_state_with_handler_on_nw_queue [C1] reporting state cancelled
default	12:46:17.209924+0200	electron-fiddle	Connection 1: destroyed
default	12:46:17.209954+0200	electron-fiddle	nw_protocol_boringssl_remove_input_handler(1012) [C1.1:2][0x7ff08d97d950] nw_protocol_boringssl_remove_input_handler forced true
default	12:46:17.209980+0200	electron-fiddle	nw_protocol_boringssl_remove_input_handler(1030) [C1.1:2][0x7ff08d97d950] Transferring nw_protocol_boringssl_t handle back into ARC for autorelease
default	12:47:47.741215+0200	runningboardd	Acquiring assertion targeting executable<electron-fiddle(501)> from originator [daemon<com.apple.coreservices.launchservicesd>:238] with description <RBSAssertionDescriptor; frontmost:4439; ID: 396-238-963; target: 4439> attributes = {
    <RBSDomainAttribute: 0x7ffb82d2bec0; domain: com.apple.launchservicesd; name: RoleUserInteractiveFocal; sourceEnvironment: 0x0>;
}
default	12:47:47.741324+0200	runningboardd	Assertion 396-238-963 (target:executable<electron-fiddle(501)>) will be created as active
default	12:47:47.742043+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Ignoring jetsam update because this process is not memory-managed
default	12:47:47.742099+0200	runningboardd	Finished acquiring assertion 396-238-963 (target:executable<electron-fiddle(501)>)
default	12:47:47.742120+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Set darwin role to: UserInteractiveFocal
default	12:47:47.742505+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Ignoring GPU update because this process is not GPU managed
default	12:47:51.300477+0200	runningboardd	Invalidating assertion 396-238-963 (target:executable<electron-fiddle(501)>) from originator 238
default	12:47:51.403445+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Ignoring jetsam update because this process is not memory-managed
default	12:47:51.403635+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Set darwin role to: UserInteractiveNonFocal
default	12:47:51.403837+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Ignoring GPU update because this process is not GPU managed
default	12:48:06.219700+0200	runningboardd	Acquiring assertion targeting executable<electron-fiddle(501)> from originator [daemon<com.apple.coreservices.launchservicesd>:238] with description <RBSAssertionDescriptor; frontmost:4439; ID: 396-238-967; target: 4439> attributes = {
    <RBSDomainAttribute: 0x7ffb850319b0; domain: com.apple.launchservicesd; name: RoleUserInteractiveFocal; sourceEnvironment: 0x0>;
}
default	12:48:06.219820+0200	runningboardd	Assertion 396-238-967 (target:executable<electron-fiddle(501)>) will be created as active
default	12:48:06.220673+0200	runningboardd	Finished acquiring assertion 396-238-967 (target:executable<electron-fiddle(501)>)
default	12:48:06.220679+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Ignoring jetsam update because this process is not memory-managed
default	12:48:06.220772+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Set darwin role to: UserInteractiveFocal
default	12:48:06.224244+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Ignoring GPU update because this process is not GPU managed
default	12:48:08.386053+0200	runningboardd	Invalidating assertion 396-238-967 (target:executable<electron-fiddle(501)>) from originator 238
default	12:48:08.489326+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Ignoring jetsam update because this process is not memory-managed
default	12:48:08.489435+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Set darwin role to: UserInteractiveNonFocal
default	12:48:08.489480+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Ignoring GPU update because this process is not GPU managed
default	12:48:45.544157+0200	runningboardd	Acquiring assertion targeting executable<electron-fiddle(501)> from originator [daemon<com.apple.coreservices.launchservicesd>:238] with description <RBSAssertionDescriptor; frontmost:4439; ID: 396-238-969; target: 4439> attributes = {
    <RBSDomainAttribute: 0x7ffb82c32580; domain: com.apple.launchservicesd; name: RoleUserInteractiveFocal; sourceEnvironment: 0x0>;
}
default	12:48:45.544264+0200	runningboardd	Assertion 396-238-969 (target:executable<electron-fiddle(501)>) will be created as active
default	12:48:45.545061+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Ignoring jetsam update because this process is not memory-managed
default	12:48:45.545133+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Set darwin role to: UserInteractiveFocal
default	12:48:45.545634+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Ignoring GPU update because this process is not GPU managed
default	12:48:45.545903+0200	runningboardd	Finished acquiring assertion 396-238-969 (target:executable<electron-fiddle(501)>)
default	12:48:48.654577+0200	hidd	Connection added: IOHIDEventSystemConnection uuid:666B671F-AC8C-4700-98B1-FCCFED3F9263 pid:4439 process:electron-fiddle type:Passive entitlements:0x0 caller:HIToolbox: _TISCopyAttachedKeyboardLanguages + 82 attributes:(null) state:0x0 events:0 mask:0x0
default	12:48:48.655795+0200	hidd	Connection removed: IOHIDEventSystemConnection uuid:666B671F-AC8C-4700-98B1-FCCFED3F9263 pid:4439 process:electron-fiddle type:Passive entitlements:0x0 caller:HIToolbox: _TISCopyAttachedKeyboardLanguages + 82 attributes:(null) state:0x1 events:0 mask:0x0
default	12:48:48.718930+0200	mDNSResponder	[R17822] DNSServiceCreateConnection STOP PID[4444](Electron Fiddle)
default	12:48:48.719374+0200	SophosScanDLegacy	process 4444 (Electron Fiddle Helper (Renderer)) exiting
default	12:48:48.719980+0200	SophosScanDLegacy	process 4443 (Electron Fiddle Helper) exiting
default	12:48:48.724838+0200	SophosScanDLegacy	process 4442 (Electron Fiddle Helper (GPU)) exiting
default	12:48:48.726769+0200	electron-fiddle	{"msg":"CLLocationManager", "event":"activity", "_cmd":"dealloc", "self":"0x7ff08b7606a0"}
default	12:48:48.726809+0200	electron-fiddle	{"msg":"state transition", "event":"state_transition", "state":"LocationManager", "id":"0x7ff08b7606a0", "property":"requestingRanging", "old":0, "new":0}
default	12:48:48.726824+0200	electron-fiddle	{"msg":"state transition", "event":"state_transition", "state":"LocationManager", "id":"0x7ff08b7606a0", "property":"updatingRanging", "old":0, "new":0}
default	12:48:48.726838+0200	electron-fiddle	{"msg":"invalidating client", "client":"0x7ff08b7543e0"}
default	12:48:48.726902+0200	electron-fiddle	{"msg":"invalidating client", "client":"0x7ff08b7543e0"}
default	12:48:48.726926+0200	electron-fiddle	{"msg":"state transition", "event":"state_transition", "state":"LocationManager", "id":"0x7ff08b7606a0", "property":"lifecycle", "old":"0x7ff08b75d6b0", "new":"0x0"}
default	12:48:48.728835+0200	hidd	Connection removed: IOHIDEventSystemConnection uuid:5B9B5135-D3CA-40F9-8952-CA1149C64418 pid:4439 process:electron-fiddle type:Monitor entitlements:0x0 caller:GameController: _GCHIDEventCallback + 96 attributes:(null) state:0x1 events:0 mask:0x0
default	12:48:48.729043+0200	hidd	Connection removed: IOHIDEventSystemConnection uuid:B5C1BF7C-87ED-4E79-AB95-629ACF16B997 pid:4439 process:electron-fiddle type:Passive entitlements:0x0 caller:HIToolbox: ___GetIOHIDEventSystemClient_block_invoke + 26 attributes:(null) state:0x1 events:0 mask:0x0
default	12:48:48.729053+0200	hidd	Connection removed: IOHIDEventSystemConnection uuid:56BD3F90-9F8A-4B07-AF02-68AD8C2A40BB pid:4439 process:electron-fiddle type:Passive entitlements:0x0 caller:HIServices: ___GetIOHIDEventSystemClient_block_invoke + 26 attributes:(null) state:0x1 events:0 mask:0x0
default	12:48:48.739418+0200	mDNSResponder	[R17805] DNSServiceCreateConnection STOP PID[4439](electron-fiddle)
default	12:48:48.738720+0200	runningboardd	Invalidating assertion 396-238-969 (target:executable<electron-fiddle(501)>) from originator 238
default	12:48:48.739149+0200	runningboardd	[executable<electron-fiddle(501)>:4439] Death sentinel fired!
```

**Comment by markwylde** [comment]
For me, (macOS M2), it never even starts:

![never-starts](https://github.com/electron/fiddle/assets/5929807/e626b871-8bfa-4329-bded-e843db8e0b08)

**Comment by dsanders11** [comment]
This may be fixed, or at least improved in v0.34.5, so I'm going to close this issue for now. If you still run into it on that version, feel free to ping me and I'll reopen.
